deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -54,19 +54,6 @@
                    (page-ref/page-ref? value)
                    (text/page-ref-un-brackets! value))
 
-                  (and
```

Comment:
This used to be the main branch for handling page links e.g. https://github.com/logseq/logseq/commit/f371a4dfac6fff81886cce57b59c87cf7c7e30f0#diff-5caa002eb84518fbfaa4afa600ecd7fc3fb59a485193f826aff2efa46cc54d97 and https://github.com/logseq/logseq/commit/88d1fa8b911102e8cca8db2c4b3537a01ca46df5. From what I can tell this branch was no longer needed once the second 'Search' branch was introduced - https://github.com/logseq/logseq/commit/2c7be3bab9e6e529cbfe8719eb2d577a164fa446

Reply:
Agreed to remove this branch.